src/riscv-double-trap.adoc

```diff
@@ -137,12 +137,12 @@ An `SRET` instruction sets the `SDT` bit to 0.
 
 [NOTE]
 ====
-A trap handler after saving the state needed for resuming from the trap,
-including `scause`, `sepc`, and `stval` among others, should clear the `SDT` bit
-when it is reentrant.
+After a trap handler, such as `scause`, `sepc`, and `stval` 
+among others, saves the state that is needed to resume from the trap, it
+should clear the `SDT` bit when it is reentrant.
```

Comment:
This change  is not right. The `scause` `sepc` are the   state  that the guideline is stating must be saved before clearing the `SDT` bit. Dont see a need to change this sentence.

Reply:
The original in this case is hard to parse though. Can we compromise with "After a trap handler saves the state, such as `scause`, `sepc`, and `stval`, that is needed to resume from the trap, it should clear the `SDT` bit...."

Reply:
We could update to:
"After a trap handler has saved the state, such as `scause`, `sepc`, and `stval`, needed for resuming from the trap and is reentrant it should clear the `SDT` bit."